### PR TITLE
Add more detailed `AIRFLOW__SCHEDULER__DAG_DIR_LIST_INTERVAL` recommendations

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/1521

### CODE REVIEW

File: astro/test-and-troubleshoot-locally.md [hunk 1]
```diff
@@ -190,9 +190,11 @@ Use the information provided here to resolve common issues with running an Astro
...
+By default, the Airflow scheduler scans the `dags` directory of your Astro project for new files every 300 seconds (5 minutes). For this reason, it might take a few minutes for new DAGs to appear in the Airflow UI. Changes to existing DAGs appear immediately. 
+
+To have the scheduler check for new DAGs more frequently, you can set the [`AIRFLOW__SCHEDULER__DAG_DIR_LIST_INTERVAL`](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#dag-dir-list-interval) environment variable to less than 300 seconds. If you have less than 200 DAGs in a Deployment, it's safe to set `AIRFLOW__SCHEDULER__DAG_DIR_LIST_INTERVAL` to `30s`.
```
Comment (jwitz) [inline astro/test-and-troubleshoot-locally.md:197]:
```suggestion
To have the scheduler check for new DAGs more frequently, you can set the [`AIRFLOW__SCHEDULER__DAG_DIR_LIST_INTERVAL`](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#dag-dir-list-interval) environment variable to less than 300 seconds. If you have less than 200 DAGs in a Deployment, it's safe to set `AIRFLOW__SCHEDULER__DAG_DIR_LIST_INTERVAL` to `30s`. See [Environment variables](environment-variables.md).
```

File: astro/test-and-troubleshoot-locally.md [hunk 2]
```diff
@@ -190,9 +190,11 @@ Use the information provided here to resolve common issues with running an Astro
...
+By default, the Airflow scheduler scans the `dags` directory of your Astro project for new files every 300 seconds (5 minutes). For this reason, it might take a few minutes for new DAGs to appear in the Airflow UI. Changes to existing DAGs appear immediately. 
+
+To have the scheduler check for new DAGs more frequently, you can set the [`AIRFLOW__SCHEDULER__DAG_DIR_LIST_INTERVAL`](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#dag-dir-list-interval) environment variable to less than 300 seconds. If you have less than 200 DAGs in a Deployment, it's safe to set `AIRFLOW__SCHEDULER__DAG_DIR_LIST_INTERVAL` to `30s`. See [Environment variables](environment-variables.md).
```
[the remark this answers is not in the document]
Comment (TJaniF) [reply astro/test-and-troubleshoot-locally.md:197]: I tried setting `AIRFLOW__SCHEDULER__DAG_DIR_LIST_INTERVAL=30s` on a repo with only a handful of DAGs and at least for me it sent my scheduler into a crash loop 👀 
`AIRFLOW__SCHEDULER__DAG_DIR_LIST_INTERVAL=30` works fine.

**Review by TJaniF — APPROVED** [review]
Two small comments otherwise LGTM :)

File: learn/airflow-scaling-workers.md [hunk 3]
```diff
@@ -66,7 +66,13 @@ Scheduler settings control how the scheduler parses DAG files and creates DAG ru
 
 - `dag_dir_list_interval`: The frequency that the DAGs directory is scanned for new files, in seconds. The lower the value, the faster new DAGs are processed and the higher the CPU usage. The default value is 300 seconds (5 minutes). 
 
-  It is helpful to know how long it takes to parse your DAGs (`dag_processing.total_parse_time`) to know what values to choose for `min_file_process_interval` and `dag_dir_list_interval`. If your `dag_dir_list_interval` is less than the amount of time it takes to parse each DAG, performance issues can occur.
+    It's helpful to know how long it takes to parse your DAGs (`dag_processing.total_parse_time`) to know what values to choose for `min_file_process_interval` and `dag_dir_list_interval`. If your `dag_dir_list_interval` is less than the amount of time it takes to parse each DAG, performance issues can occur. 
+  
+  :::tip
+
+  If you have less than 200 DAGs in a Deployment on Astro, it's safe to set `AIRFLOW__SCHEDULER__DAG_DIR_LIST_INTERVAL=30s` as a Deployment-level [environment variable](https://docs.astronomer.io/astro/environment-variables.md).
```
[the remark this answers is not in the document]
Comment (kentdanas) [reply learn/airflow-scaling-workers.md:73]: Is the interval always in seconds? Or is the `s` needed on the variable itself (like if you can also provide `m` for in minutes)? (I think it's the former, but not totally sure).

**Review by paolaperaza — APPROVED** [review]

File: learn/airflow-scaling-workers.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (TJaniF) [reply learn/airflow-scaling-workers.md:73]: Yes, it is the former.

**Pull request merged**